packages/shared/catalogs/sagas.ts

```diff
@@ -54,9 +63,39 @@ const BASE_BUILDER_DOWNLOAD_URL = `${BUILDER_SERVER_URL}/storage/contents`
  *
  */
 export function* catalogsSaga(): any {
-  yield takeEvery(WEARABLES_REQUEST, handleWearablesRequest)
+  yield takeEvery(WEARABLES_REQUEST, handleItemRequest)
   yield takeEvery(WEARABLES_SUCCESS, handleWearablesSuccess)
   yield takeEvery(WEARABLES_FAILURE, handleWearablesFailure)
+  yield takeEvery(EMOTES_REQUEST, handleItemRequest)
+  yield takeEvery(EMOTES_SUCCESS, handleWearablesSuccess)
+  yield takeEvery(EMOTES_FAILURE, handleWearablesFailure)
+}
+
+export function* handleItemRequest(action: EmotesRequest | WearablesRequest) {
+  const { filters, context } = action.payload
+
+  const valid = areFiltersValid(filters)
+  if (valid) {
+    try {
+      const fetchContentServer: string = yield select(getFetchContentServer)
+
+      const response: PartialWearableV2[] = yield call(fetchItemsFromCatalyst, action, filters)
+      const net: ETHEREUM_NETWORK = yield select(getSelectedNetwork)
+      const assetBundlesBaseUrl: string = getAssetBundlesBaseUrl(net) + '/'
+
+      const v2Wearables: WearableV2[] = response.map((wearable) => ({
+        ...wearable,
+        baseUrl: wearable.baseUrl ?? fetchContentServer + '/contents/',
+        baseUrlBundles: assetBundlesBaseUrl
+      }))
+
+      yield put(wearablesSuccess(v2Wearables, context))
```

Comment:
this is always dispatching a `wearableSuccess` even for an `emoteRequest` (instead of an `emoteSuccess`), is this intended?

Reply:
No! It should dispatch one of the new ones, updated! thanks for the catch :)